app/src/androidTest/java/com/example/vybe/MyVibesActivityTest.java

```diff
@@ -350,16 +327,11 @@ public void Test04_AddVibeEvent_AllFields() throws InterruptedException, UiObjec
 
         // Check details of vibe are correct
         onView(withId(R.id.view_date_text_view)).check(matches(withText(containsString(formatter.format(date2).split(" ")[0]))));
-        onView(withId(R.id.view_reason_text_view)).check(matches(withText(containsString("I am Sad"))));
+        onView(withId(R.id.view_reason_text_view)).check(matches(withText(containsString("I am Scared"))));
         onView(withId(R.id.view_soc_sit_text_view)).check(matches(withText(containsString(SocSit.ALONE.getDesc()))));
 
-        // Check we can see the map view of the location
-        onView(withId(R.id.view_vibe_map))
-                .check(matches(isDisplayed()));
-
     }
 
-    // TODO: Get dates of two items and compare their dates to verify ordering
```

Comment:
this test doesn't actually test the dates all it does is check the month is on the screen